dashboard/db/migrate/20170505231542_remove_name_from_plc_course.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveNameFromPlcCourse < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :plc_courses, :name
```

Comment:
I just realized (when I accidentally tried rolling this back in dev) that this is not reversible. It displays this:

```
== 20170505231542 RemoveNameFromPlcCourse: reverting ==========================
rake aborted!
StandardError: An error has occurred, all later migrations canceled:



remove_column is only reversible if given a type.
```

It should be 
```ruby
remove_column :plc_courses, :name, :string
```